src/state/schedule-data/schedule-condition/revision-info.reducer.ts

```diff
@@ -36,13 +36,11 @@ export function revisionInfoReducer(
   state: RevisionType = "actual",
   action: ActionModel<RevisionType>
 ): RevisionType {
-  switch (action.type) {
-    case RevisionReducerAction.CHANGE_REVISION:
-      if (!action.payload) {
-        return state;
-      }
-      return action.payload;
-    default:
+  if (action.type === RevisionReducerAction.CHANGE_REVISION) {
+    if (!action.payload) {
       return state;
+    }
+    return action.payload;
   }
+  return state;
```

Comment:
To nie powinno być przepisane zgodnie z `redux=best-practices`?

Reply:
Done